maturity-level-one/src/README.md

```diff
@@ -47,6 +47,7 @@ This implementation includes:
 ## Document your API
 	* Swagger generation with OperationId, Xml comments, Swagger attributes
 	* [ApiExplorerSettings(IgnoreApi = true)] used to exclude operations from OpenAPI output
+	* Consider to introduce a SwaggerOperationFilter to have more control on the consumed/produced media types. [SwaggerConsumesProduces(Consumes="application/json, text/json", Produces="application/json", Clear=true)]
```

Comment:
If this isn't supported out of the box already, please consider creating an issue on https://github.com/arcus-azure/arcus.webapi to add it there.

Otherwise people need to copy paste it every time.

Reply:
Produces and Consumes are automatically generated according to the message formatters.

This attribute can be used to add a single media type (Clear=false) or to fully replace produces and consumes (Clear=true)
This attribute helps to explicitly define what is expected and what is produced when you are not satisfied by the out of the box generation.
For example,  I wanted to use this attribute to fix the update operation done via json-patch

_Out of the box YAML_
      consumes:
        - application/json-patch+json
        - application/json
        - text/json
        - application/*+json
      produces: []

_What i wanted_
      consumes:
        - application/json-patch+json
      produces: 
        - application/json

Of course if limit the content negotiation, a 415 should be returned

Reply:
Maybe more in 2 than 1?

Reply:
Content negotiation tracked with #86

Reply:
Swagger Produces / Consumes attributes have been introduced in swashbuckle ver 5.0 (beta version as today).
The v5.0 produces open api v3.0 - In this maturity level we will stay stick to the version 2.0 (swashbuckle 4.0.1)